Sources/Apollo/GraphQLExecutor.swift

```diff
@@ -24,7 +24,7 @@ struct GraphQLResolveInfo {
   }
 }
 
-struct GraphQLResultError: Error, LocalizedError {
+public struct GraphQLResultError: Error, LocalizedError {
   let path: ResponsePath
   let underlying: Error
```

Comment:
Makes sense -worth making `path` and `underlying` public as well?

Reply:
Well spotted! It's definitely worth it 👍

Reply:
Now that I see, that will require to also make `ResponsePath` public. Is that ok?

Reply:
Ehhh let's keep that private but I think making the `underlying` error public probably wouldn't hurt.

Reply:
I agree 👌